dungeon/chain.json

```diff
@@ -0,0 +1,49 @@
+{
+  "chain_name": "dungeon",
+  "status": "live",
+  "network_type": "mainnet",
+  "chain_type": "cosmos",
+  "website": "https://www.cryptodungeon.org",
+  "pretty_name": "Dungeon Chain",
+  "chain_id": "dungeon-1",
+  "bech32_prefix": "dungeon",
```

Comment:
Please register this prefix here: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
it was registered at SLIP-173 as 'dgn', but here it has 'dungeon'